src/main/scala/org/broadinstitute/dsde/firecloud/core/ProfileClient.scala

```diff
@@ -332,7 +336,9 @@ class ProfileClientActor(requestContext: RequestContext) extends Actor with Fire
     RequestComplete(NoContent)
   }.recover {
     // intentionally quash errors so as not to leak sensitive data
-    case _ => RequestCompleteWithErrorReport(InternalServerError, "Error synchronizing NIH whitelist")
+    case _ =>
+      log.error("Error synchronizing NIH whitelist")
+      RequestCompleteWithErrorReport(InternalServerError, "Error synchronizing NIH whitelist")
```

Comment:
@davidangb - I'm seeing this error case trigger regularly while testing. I don't see why yet, so will be digging into that problem.